### PR TITLE
Update CI, pin more versions

### PR DESCRIPTION
**Pull request by nrbgt**
[Binder](https://mybinder.org/v2/gh/nrbgt/ipyradiant/gh-75-update-ci?urlpath=lab%2Ftree%2Fexamples%2FGraph_Explorer.ipynb)

- includes #75
  - this can either get merged in there, or we wait for that to land, then merge this so it can be better reviewed (a lot of linter noise, but not that much stuff)
- [x] uses mamba(forge)
- [x] conda-forge only 
- [x] upgrade linters (lots of paint)
- [x] pins more versions (starting to get lab3 drift)

### CODE REVIEW

**Review by lnijhawan — APPROVED** [review]

File: src/ipyradiant/visualization/interactive_exploration.py [hunk 1]
```diff
@@ -89,21 +93,35 @@ class GetOutgoingPredicateObjects(SPARQLQueryFramer):
 
     """
 
-
-# Throughout this class, we assign the layout to self.cyto_graph_layout multiple times.
-# This is so that the graph refreshes the layout every time nodes are added or removed,
-# which provides an optimal viewing experience.
+    @classmethod
```
Comment (zwelz3) [inline src/ipyradiant/visualization/interactive_exploration.py]: This is not in master. I think it got added back during merge conflict resolution
https://github.com/jupyrdf/ipyradiant/blob/master/src/ipyradiant/visualization/interactive_exploration.py

File: .ci/environment.yml [hunk 2]
```diff
@@ -2,8 +2,6 @@ name: ipyradiant-base
 
 channels:
   - https://conda.anaconda.org/conda-forge
-  - https://conda.anaconda.org/anaconda
-  - https://conda.anaconda.org/msys2
```
Comment (zwelz3) [inline .ci/environment.yml:6]: Do we need `nodefaults` here too?

File: src/ipyradiant/visualization/interactive_exploration.py [hunk 3]
```diff
@@ -75,7 +78,8 @@
 
 class GetOutgoingPredicateObjects(SPARQLQueryFramer):
     """
-    Return all triples for non-Literal objects (and the optional object labels).
```
Comment (zwelz3) [inline src/ipyradiant/visualization/interactive_exploration.py]: This is not in master. I think it got added back during merge conflict resolution
https://github.com/jupyrdf/ipyradiant/blob/master/src/ipyradiant/visualization/interactive_exploration.py

File: src/ipyradiant/visualization/interactive_exploration.py [hunk 4]
```diff
@@ -89,21 +93,35 @@ class GetOutgoingPredicateObjects(SPARQLQueryFramer):
...
+            **initBindingsKwarg,
+        )
+        # Validating with known requirements on query results
+        # TODO
+        return qres
 
 
 class InteractiveViewer(W.VBox):
     expand_button = trt.Instance(W.Button)
-    undo_button = trt.Instance(W.Button)
```
Comment (zwelz3) [inline src/ipyradiant/visualization/interactive_exploration.py]: See above,. I think this whole file needs to be re-cloned from master.

**Review by zwelz3 — CHANGES_REQUESTED** [review]
Minor issue with merge conflict resolution.

File: .ci/environment.yml (hunk 2 above)
Comment (nrbgt) [reply .ci/environment.yml:6]: good call.

File: src/ipyradiant/visualization/interactive_exploration.py (hunk 1 above)
Comment (nrbgt) [reply src/ipyradiant/visualization/interactive_exploration.py]: c020b69

File: .ci/environment.yml (hunk 2 above)
Comment (nrbgt) [reply .ci/environment.yml:6]: 598b578ca0

**Review by zwelz3 — APPROVED** [review]

**Pull request merged**